### PR TITLE
Slashing import

### LINKED ISSUE #296: [Add validator] Keystore upload
Keystore upload step should additionally contain:
- checkbox "I'm switching validator" which displays additional required upload field for slashing protection json
- optional field validator name

### LINKED ISSUE #297: [Add validator] Ask user to change password after keystore upload
In step after keystore upload, user should have option to pick between creating new password (fields old_password, new_password, confirm_password) and skipping (keeping old password),

Upon completition, user should be taken to dashboard where new validator is going to be added on top of list.

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #296 
closes #297

### CODE REVIEW

File: src/renderer/containers/Onboard/CreatePassword/CreatePasswordContainer.tsx [hunk 1]
```diff
@@ -52,28 +49,26 @@ export class CreatePassword extends Component<Pick<RouteComponentProps, "history
...
-        }
+            },
+        ];
 
         const {errorMessages, loading} = this.state;
         return (
             <>
-                <h1>{!this.props.path ? "Create" : "Validate"} a password</h1>
-                {!this.props.path && <p>You will use this password to unlock applications and keys.</p>}
+                <h1>{!this.props.path ? "Create" : "Change"} a password</h1>
```
Comment (morrigan) [inline src/renderer/containers/Onboard/CreatePassword/CreatePasswordContainer.tsx:70]: maybe it's better to pass a real prop here? It's confusing condition and used multiple times through file

Also, "Create a password" and "Change the password" :D

File: src/renderer/containers/Onboard/CreatePassword/CreatePasswordContainer.tsx [hunk 2]
```diff
@@ -52,28 +49,26 @@ export class CreatePassword extends Component<Pick<RouteComponentProps, "history
...
+                <p>You will use this password to unlock applications and keys.</p>
```
Comment (morrigan) [inline src/renderer/containers/Onboard/CreatePassword/CreatePasswordContainer.tsx:71]: "...unlock validator account" might be better wording

File: src/renderer/containers/Onboard/SigningKey/Import/FileUploadImport.tsx [hunk 3]
```diff
@@ -15,6 +17,8 @@ export const FileUploadImport: FC<IOwnProps> = ({history}) => {
     const [path, setPath] = useState<null | string>(null);
     const [fileName, setFileName] = useState<null | string>(null);
     const [publicKey, setPublicKey] = useState("");
+    const [checked, setChecked] = useState(false);
+    const [isShowingPromp, setShowingPromp] = useState(false);
```
Comment (morrigan) [inline src/renderer/containers/Onboard/SigningKey/Import/FileUploadImport.tsx:21]: maybe use better naming "displayPasswordConfirmation"

File: src/renderer/containers/Onboard/SigningKey/Import/FileUploadImport.tsx [hunk 4]
```diff
@@ -36,40 +40,60 @@ export const FileUploadImport: FC<IOwnProps> = ({history}) => {
...
                 />
-                <label htmlFor='file' className={error && "error"}>
-                    <FontAwesomeIcon icon={faUpload} transform={{x: -10}} />
-                    {!fileName ? "Choose a file..." : fileName}
-                    {error && <FontAwesomeIcon icon={faExclamationTriangle} className='error-icon' size='lg' />}
-                </label>
-                <div className={"error-message error-message-wide"} style={{marginLeft: "28px"}}>
-                    {error}
-                </div>
+                <CheckBox checked={checked} label="I'm switching validator" id='slashing' onClick={onCheckboxClick} />
```
Comment (morrigan) [inline src/renderer/containers/Onboard/SigningKey/Import/FileUploadImport.tsx:82]: "I'm switching validator software" or "I'm switching validator client"

File: src/renderer/containers/Onboard/SigningKey/Import/SlashingUploadImport.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        setError("");
+        const filePath = event.target.files[0]?.path;
+        setFileName(event.target.files[0].name);
+        if (!filePath) {
+            setFileName(null);
+            setError("Please select a file");
+        } else if (validateSlashingFile(filePath)) {
+            setPath(filePath);
+        } else {
+            setError("File is incorrect, try again whit different file");
```
Comment (morrigan) [inline src/renderer/containers/Onboard/SigningKey/Import/SlashingUploadImport.tsx:29]: with*

File: src/renderer/services/jsonValidation/index.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import Ajv, {Options} from "ajv";
+import slashingJson from "./slashing.schema.json";
+import {readFileSync} from "fs";
+
+export const createValidator = (schema: object, options: Options = {}) => (data: object | string): boolean =>
```
Comment (morrigan) [inline src/renderer/services/jsonValidation/index.ts:5]: This is tricky naming as Validator is something else in our case :D :D

**Review by morrigan — COMMENTED** [review]
LGTM, just left out minor suggestions

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**